src/openrct2/ui/UiContext.h

```diff
@@ -117,6 +117,7 @@ namespace OpenRCT2
             virtual void OpenURL(const std::string& url) abstract;
             virtual std::string ShowFileDialog(const FileDialogDesc& desc) abstract;
             virtual std::string ShowDirectoryDialog(const std::string& title) abstract;
+            virtual bool HasFilePicker() abstract;
```

Comment:
```suggestion
            virtual bool HasFilePicker() const abstract;
```
If the state of the object/class is not modified the functions should be marked const